src/game/option_sound.c

```diff
@@ -102,6 +111,16 @@ void Option_Sound(INVENTORY_ITEM *inv_item)
             Music_SetVolume(g_Config.music_volume);
             Sound_Effect(SFX_MENU_PASSPORT, NULL, SPM_ALWAYS);
         }
+
+        if (g_Config.music_volume == 0) {
+            Text_Hide(m_Text[TEXT_LEFT_ARROW], true);
+        } else if (g_Config.music_volume == 10) {
```

Comment:
Can we create a define for `10` and update existing usages?
Maybe even for `0` as well

`MIN_VOLUME` `MAX_VOLUME`